docs/API.md

```diff
@@ -392,13 +393,23 @@ Sometimes additional keys are present.
 This key is added when the module name is `"NAMESERVER"`.
 
 
-### Timestamp
+### Timestamp (deprecated)
 
 Basic data type: string
 
+**Deprecated representation** (planned removal: v2023.1).
```

Comment:
Maybe add an empty line after this to make the rendering nicer?

Reply:
I agree.

Reply:
added